src/Buttons.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { fireEvent, render } from '@testing-library/react';
+
+import Buttons from './Buttons';
+
+describe('Buttons', () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const handleClickSelect = jest.fn();
+
+  it('버튼을 클릭하면 handleClickSelect 함수가 출력됩니다.', () => {
+    const { getByText } = render((
+      <Buttons
+        buttonNameList={['서울']}
+        handleClickSelect={handleClickSelect}
+      />));
+
+    const button = getByText('서울');
+
+    expect(handleClickSelect).not.toBeCalled();
+    fireEvent.click(button);
+    expect(handleClickSelect).toBeCalled();
+  });
+
+  context('ButtonsNameList가 주어지면,', () => {
+    const buttonNameList = ['서울', '대전', '대구', '부산', '광주', '강원도', '인천'];
+
+    it('ButtonsNameList의 이름을 가진 버튼들이 출력됩니다.', () => {
+      const { container } = render(<Buttons buttonNameList={buttonNameList} />);
+
+      buttonNameList.forEach((buttonName) => {
+        expect(container).toHaveTextContent(buttonName);
+      });
+    });
+  });
+});
```

Comment:
![image](https://user-images.githubusercontent.com/14071105/96604873-8d01de80-1330-11eb-9e3a-56136c288473.png)

빈 줄을 추가해야하는 이유는 GitHub의 컨벤션 때문에 그렇습니다. GitHub은 변경사항을 표시할 때 변경전 부분과 변경 후 를 모두 보여줍니다. 

```js
console.log('Hello');
```

만약 위와 같이 빈 줄이 없는 코드에서 아래에 코드를 추가하게 되면 Git은 다음과 같이 표시합니다.

```js
- console.log('Hello');
+ console.log('Hello');
+ console.log('world!');
```

저는 단순히 아래 한 줄만 추가했는데 왜 세줄이 변경되었다고 나오는 걸까요? 그 이유는 줄바꿈 문자 \n가 들어갔기 때문입니다. 

```js
- console.log('Hello');
+ console.log('Hello');\n
+ console.log('world');
```

만약 미리 빈 줄을 하나 추가한 상태에서 아래에 새로운 코드를 작성할 경우에는 다음과 같이 보여집니다.

```js
+ console.log('world');
```

이미 \n이 추가되어 있기 때문이죠. 따라서 코드 마지막 줄에 빈 줄 하나를 항상 추가해주는게 좋습니다.

```suggestion
});
// 새로운 코드
```

Reply:
그렇군요. 그래서 eslint에서 밑줄을 추가하라고 나오는 거였군요. 감사합니다.